### ISSUE TITLE
Smite Mouse movement 

### ISSUE THREAD

**Issue by Sinnabag** [fqlx/XboxKeyboardMouse#172]
Ingame its very choppy tried adjusting it alot i dunno very much about this program maby someone can help me out

**Comment by DavidRieman** [comment]
Have you followed the dead-zone guide?
 https://github.com/fqlx/XboxKeyboardMouse#dead-zones

**Comment by Sinnabag** [comment]
Yes i actually tried mutiple things on top of whats in there like setting to the max setting which in smite is 60/60 with both deadzone being 10/10 then tried putting both dead zones on 1/10 which didnt work the issue most of the time is when i move the mouse it moves in like pixels or very skippy in sense

**Comment by Sinnabag** [comment]
Also tried putting it in the middle with the sens and it still has the same effect i might be able to get a video of it its helpful

**Comment by gamerontheline79** [comment]
anytime someone brings this issue up they like to just push it away. Im having the same exact issue and have set everything up the way they say to. Nothing
